### PR TITLE
GDB-10578 Add notes regarding the Minikube multinode setup

### PR DESCRIPTION
**Pull request by cloudvlad**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # GraphDB Helm chart release notes
 
+## Version 11.1.3
```
Comment (mihailradkov) [inline CHANGELOG.md:3]: Rebase the branch on top of `PRO-444-features-for-11-2` and include this for `11.2.0` in the changelog. We don't want to release a patch version for a minor change in the readme.

File: README.md [hunk 2]
```diff
@@ -638,6 +638,13 @@ DNS is broken. This is a common issue with Minikube between system restarts or w
 Minikube driver is used. Please refer to
 https://kubernetes.io/docs/tasks/administer-cluster/dns-debugging-resolution/.
 
+**Filesystem provisioning errors (in Minikube)**
+
+When expanding your Minikube cluster from one to two or more nodes, you need to adjust the
```
Comment (mihailradkov) [inline README.md:643]: It's not clear what is the issue here.

File: README.md [hunk 3]
```diff
@@ -651,6 +651,16 @@ DNS is broken. This is a common issue with Minikube between system restarts or w
 Minikube driver is used. Please refer to
 https://kubernetes.io/docs/tasks/administer-cluster/dns-debugging-resolution/.
 
+**Filesystem provisioning errors (in Multi-Node Minikube Cluster)**
+
+When expanding your Minikube cluster from one to two or more nodes to deploy different GraphDB
+instances across multiple nodes to ensure high availability, you may encounter errors when
```
Comment (ivorusev) [inline README.md:657]: What errors and where would be useful to know.

**Review by ivorusev — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (mihailradkov) [reply README.md:657]: Even we are not really sure what is going on, so let's leave it like this for now.

**Review by mihailradkov — APPROVED** [review]

**Pull request merged**